ArmaReforgerServerTool/Models/ServerConfiguration.cs

```diff
@@ -132,15 +166,42 @@ public Game(string name, string password, string passwordAdmin, string[] admins,
             this.mods = mods;
         }
 
-        public static Game Default => new(string.Empty, string.Empty, string.Empty, Array.Empty<string>(), 
-            string.Empty, 0, false, false, Array.Empty<string>(), GameProperties.Default, Array.Empty<Mod>());
+        public static Game Default => new(
+            string.Empty,
+            string.Empty,
+            string.Empty,
+            Array.Empty<string>(),
+            string.Empty,
+            DEFAULT_PLAYERS,
+            DEFAULT_VISIBLE,
+            DEFAULT_CROSS_PLATFORM,
+            Array.Empty<string>(),
+            GameProperties.Default,
+            Array.Empty<Mod>()
+        );
     }
 
     /// <summary>
     /// Structure representing the gameProperties block of the Server Config
     /// </summary>
     public class GameProperties
     {
+        public static readonly int MIN_SERVER_VIEW_DISTANCE = 500;
```

Comment:
this is a general comment for all changes in this, with assignments, I think it looks nicer to put the = inline.

E.g.:

```cs
 public static readonly int MAX_SERVER_VIEW_DISTANCE     = 10000;
 public static readonly int DEFAULT_SERVER_VIEW_DISTANCE = 1600;
 public static readonly int MIN_SERVER_GRASS_DISTANCE    = 0;
 public static readonly int MAX_SERVER_GRASS_DISTANCE    = 150;
```